vars/tagDeployment.groovy

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env groovy
 
-def call(String microservice, String aws_profile = "test", String tag = null) {
-  tag = tag ?: gitCommit()
-
-  build job: 'run-tag-and-capture-notes-commit-based',
-    parameters: [
-      string(name: 'ENVIRONMENT', value: aws_profile),
-      string(name: 'COMMIT_HASH', value: tag),
-      string(name: 'SERVICE_TO_TAG', value: microservice)
-    ]
+def call() {
+  commit_hash = gitCommit()
+
+  date = new java.text.SimpleDateFormat("yyyy-MM-dd-HH:mm:ss").format(new Date())
+
+  latest_tag_components = "git describe --abbrev=0 --match 'alpha_release-*'".execute.text.split('-')
```

Comment:
won't `"git describe --abbrev=0 --match 'alpha_release-*'".split('-')` work? not sure why we need the `.execute.text`

Reply:
ok i get it now. `"git describe --abbrev=0 --match 'alpha_release-*'"` is the "script" that needs to be executed.